.github/workflows/runE2ETest.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
         nodeVersion:
-          - 16.15.1
+          - 18.11.0
```

Comment:
can we remove the matrix thing since it's just the one version and use the nvmrc to pick the right version?       - ```uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'``` from `buildAll.yml`

Reply:
actually nevermind ...with the duel repo thing it gets complicated